app/views/sales/_salelist.html.erb

```diff
@@ -0,0 +1,65 @@
+<% require 'date' %>
+
+<% @month_num = {"01" => 'JAN', "02" => 'FEB', "03" => 'MAR', "04" => 'APR', "05" => 'MAY', "06" => 'JUN', "07" => 'JUL', "08" => 'AUG', "09" => 'SEP', "10" => 'OCT', "11" => 'NOV', "12" => 'DEC'} %>
```

Comment:
This instance variable definition also exists in the controller.  It shouldn't be defined in two places.